wezterm-gui/src/termwindow/mouseevent.rs

```diff
@@ -667,7 +667,18 @@ impl super::TermWindow {
             Some(MouseCapture::TerminalPane(_))
         );
 
-        for pos in self.get_panes_to_render() {
+        let panes = if let Some(float_pane) = self.get_float_pane_to_render() {
+            if position.column >= float_pane.left && position.column <= float_pane.left + float_pane.width
+                && position.row as usize >= float_pane.top && position.row as usize <= float_pane.top + float_pane.height {
+                vec![float_pane]
+            } else {
+                return
```

Comment:
does this return prevent mouse handling from working in the other panes when a float is present?
Is that intentional? If so, please add a comment around this block to explain this, as it isn't clear from the code.

Also, I prefer an early return style to avoid lots of nested blocks, something like this:

```rust
let mouse_in_float = position.column >= float_pane.left && 
    position.column <= float_pane.left + float_pane.width &&
    position.row as usize >= float_pane.top &&
    position.row as usize <= float_pane.top + float_pane.height;

if !mouse_in_float {
  // Mouse events are not dispatched to the other panes when
  // a floating pane is active, because REASON
  return;
}
vec![float_pane]
```